tsconfigbase.json

```diff
@@ -16,7 +16,7 @@
     "resolveJsonModule": true,
     "sourceMap": true,
     "strictNullChecks": true,
-    "target": "es2017",
+    "target": "es2018",
```

Comment:
@afshin are we ok to change that for a minor release?

Reply:
I am not sure I fully understand the ramifications. I don’t *think* this is backward-incompatible but I’m not 100% confident.

Reply:
This is backward incompatible in the sense that it raises the required version of web browsers.

Reply:
I think in that sense we might be okay because all the [browsers we support are covered](
https://caniuse.com/?feats=mdn-javascript_builtins_regexp_dotall,mdn-javascript_builtins_regexp_lookbehind_assertion,mdn-javascript_builtins_regexp_named_capture_groups,mdn-javascript_builtins_regexp_property_escapes,mdn-javascript_builtins_symbol_asynciterator,mdn-javascript_functions_method_definitions_async_generator_methods,mdn-javascript_grammar_template_literals_template_literal_revision,mdn-javascript_operators_destructuring_rest_in_objects,mdn-javascript_operators_spread_spread_in_destructuring,promise-finally).

Do you think we have a problem?